OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -371,6 +371,10 @@ bool TracerThread::OpenTracepoints(const std::vector<int32_t>& cpus) {
       "task", "task_rename", cpus, &tracing_fds_, &task_rename_ids_,
       &tracepoint_ring_buffer_fds_per_cpu, &ring_buffers_);
 
+  tracepoint_event_open_errors |= !OpenRingBuffersForTracepoint(
```

Comment:
Chatted offline: this is not activating the collection of the data for the tracepoints yet - so there is no performance impact due to this change.

Reply:
It does and this causes big performance problems because of the `LOG` in `ProcessSampleEvent`. If we want this experimental code to be already in, it should be behind a flag.